scripts/cov.py

```diff
@@ -0,0 +1,20 @@
+import os
+import re
+import sys
+
+# read output of (potentially many invocations of) "go test -cover -short",
+# e.g., "coverage: 65.7% of statements"
+cover_report_lines = sys.stdin.read()
+
+if len(cover_report_lines) == 0:
```

Comment:
is there a possibility to print some error condition here ? how does the output know of this specific failure?

Reply:
This script outputs nothing, it returns an exit code.  If it's important to differentiate we could exit with a different code.

Reply:
Done in 1faca96